snapshot-labs/sx-monorepo
# feat: add statements to delegates list

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/174

This PR adds a new `statement` column in the space delegates list, as well as a sub-menu for each row, with a link to:

- open the modal delegate, prefilled with the delegatee address
- the delegatee on the block explorer
- its space user profile

### Note

- install a new `remove-markdown` dependency to strip all markdown code from statement excerpt

### How to test

1. Go to http://localhost:8080/#/matic:0x80D0Ffd8739eABF16436074fF64DC081c60C833A/delegates
2. There should be a new `Statement` column, showing the delegatee statement on 2 lines maximum
3. A new `...` button, on each line far right, with a dropdown menu
4. The "Delegate" menu should open the delegate modal, prefilled with the delegatee address
5. The `View profile` menu should redirect to this delegatee space user profile
6. The `See on block explorer` should open this delegatee address on the block explorer
